src/ml/neural_net/mps_layer_instance_norm_data_loader.h

```diff
@@ -0,0 +1,92 @@
+#ifndef mps_layer_instance_norm_data_loader_h
+#define mps_layer_instance_norm_data_loader_h
+
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h> 
+
+typedef struct{
+  int size;
+  float beta1;
+  float beta2;
+  float epsilon;
+  float lr_t;
+} params_t;
+
+API_AVAILABLE(macos(10.14))
+@interface InstanceNormDataLoader: NSObject <MPSCNNInstanceNormalizationDataSource> {
```

Comment:
Objective C class names must be prefixed to avoid collisions, e.g. with something like `TCMPS`. Please apply this rule to all future PRs.